website/docs/blueprints/data-analytics/ray-data-processing.md

```diff
@@ -0,0 +1,681 @@
+- **Output Path**: `s3://bucket/iceberg-warehouse/` - Contains structured Parquet files
+- **Storage Format**: Iceberg uses efficient columnar Parquet format with metadata for ACID transactions
+:::
+
+In AWS S3 Console, it should looks like below:
+
+![s3](img/raydata-s3-struc.png)
+
+
+### Log Snippet After RayData Processing
+
+Here's how the data transformation looks before and after Ray Data processing:
+
+<Tabs>
+<TabItem value="before" label="Before Processing (Raw S3 JSON)">
+
+**Raw Fluent Bit logs in S3** - Each log line wrapped in JSON with redundant metadata:
+
+```json
+{
+  "log": "2024-01-15 14:23:45 INFO SparkContext: Running Spark version 3.5.0\n",
+  "stream": "stdout",
+  "time": "2024-01-15T14:23:45.123456Z",
+  "kubernetes": {
+    "pod_name": "spark-driver-abc123",
+    "namespace_name": "spark-team-a",
+    "pod_id": "12345678-1234-1234-1234-123456789012",
+    "labels": {
+      "spark-role": "driver",
+      "spark-app-id": "spark-application-12345"
+    },
+    "container_name": "spark-driver",
+    "container_image": "spark:3.5.0"
+  }
+}
+{
+  "log": "2024-01-15 14:23:46 ERROR TaskSchedulerImpl: Lost executor 1: Container killed\n",
+  "stream": "stderr",
+  "time": "2024-01-15T14:23:46.234567Z",
+  "kubernetes": {
+    "pod_name": "spark-executor-def456",
+    "namespace_name": "spark-team-a",
+    "labels": {
+      "spark-role": "executor",
+      "spark-app-id": "spark-application-12345"
+    }
+  }
+}
+```
+
+</TabItem>
+<TabItem value="after" label="After Processing (Structured Iceberg)">
+
+**Processed data in Apache Iceberg** - Clean, structured schema optimized for querying:
+
+```sql
+-- Query the processed data
+SELECT * FROM raydata_spark_logs.spark_logs LIMIT 2;
+```
+
+| timestamp | log_level | component | message | pod_name | namespace | spark_role | application_id |
+|-----------|-----------|-----------|---------|----------|-----------|------------|----------------|
+| 2024-01-15 14:23:45 | INFO | SparkContext | Running Spark version 3.5.0 | spark-driver-abc123 | spark-team-a | driver | spark-application-12345 |
+| 2024-01-15 14:23:46 | ERROR | TaskSchedulerImpl | Lost executor 1: Container killed | spark-executor-def456 | spark-team-a | executor | spark-application-12345 |
+
+**✅ Key Improvements:**
+- **Structured Fields** - Easy to query specific log levels, components, and time ranges
+- **Deduplicated Metadata** - No redundant Kubernetes metadata per log line
+- **Columnar Storage** - Efficient storage and query performance with Parquet format
+- **Schema Evolution** - Add new fields without breaking existing queries
+- **ACID Transactions** - Consistent reads even during concurrent processing
+
+</TabItem>
+</Tabs>
+
+### Option 1: Query Iceberg Tables
+
+Use the built-in data verification script provided in the blueprint that automatically sets up a Python virtual environment and all required dependencies:
+
+ from your environment.
+
+```bash
+# Make script executable (first time only)
```

Comment:
Add here saying that...Update <ENTER_S3_BUCKET> and <AWS_REGION> in the shell script